app/webpacker/components/EditSchedule/EditVenues/VenuePanel.js

```diff
@@ -36,7 +36,9 @@ function VenuePanel({
   const confirm = useConfirm();
 
   const handleCoordinateChange = (evt, { name, value }) => {
-    dispatch(editVenue(venue.id, name, toMicrodegrees(value)));
+    dispatch(
+      editVenue(venue.id, name, Number.isNaN(toMicrodegrees(value)) ? 0 : toMicrodegrees(value)),
```

Comment:
Could this logic move into the `toMicrodegrees` function?
I understand the reason behind your approach, and your solution is fine except for the fact that right now it's too densely packed into one line. (And you're calling `toMicrodegrees` twice, that seems redundant)

Reply:
If you find a better place for the input sanitization, feel free. I just don't want it to happen right inside the setter fn call